<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>工具箱</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>进程工具</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>368</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>检测类型</string>
     </property>
     <widget class="QRadioButton" name="radioButton_1">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>24</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>108</x>
        <y>23</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>进程</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>22</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>计划任务</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>67</y>
       <width>368</width>
       <height>38</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_1">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>7</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>内容值</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_1">
      <property name="geometry">
       <rect>
        <x>57</x>
        <y>9</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_3">
      <property name="geometry">
       <rect>
        <x>317</x>
        <y>8</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="button_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>117</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭端口</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>152</y>
       <width>368</width>
       <height>146</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="button_1">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>117</y>
       <width>88</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>检测端口</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>实用工具</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>-1</y>
       <width>391</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>加密解密</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_encrypt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>361</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>加密解密工具</string>
       </property>
       <widget class="QLabel" name="label_input">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>23</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>输入内容</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_origin">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>24</y>
          <width>281</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="button_md5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>54</y>
          <width>88</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>计算MD5</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_md5">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>55</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="button_sha1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>86</y>
          <width>88</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>计算SHA1</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_sha1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>88</y>
          <width>241</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_input_4">
        <property name="geometry">
         <rect>
          <x>14</x>
          <y>115</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>base64互转</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_base64_1">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>140</y>
          <width>321</width>
          <height>41</height>
         </rect>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="button_base64_1">
        <property name="geometry">
         <rect>
          <x>58</x>
          <y>188</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>加密↓</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_base64_2">
        <property name="geometry">
         <rect>
          <x>209</x>
          <y>188</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>解密↑</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_base64_2">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>220</y>
          <width>321</width>
          <height>41</height>
         </rect>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>模型生成</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox_encrypt_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>361</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>Flask模型代码生成</string>
       </property>
       <widget class="QLabel" name="label_input_5">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>23</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>连接字符串</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_connect">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>326</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>mysql+pymysql://username:password@host/database</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_gen_model">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>79</y>
          <width>81</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>生成代码</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_model_code">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>110</y>
          <width>331</width>
          <height>151</height>
         </rect>
        </property>
        <property name="plainText">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>其他工具</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
